climada/test/test_api_client.py

```diff
@@ -35,38 +35,40 @@ class TestClient(unittest.TestCase):
 
     def test_data_type(self):
         """"""
-        lpdt = Client().get_data_type("litpop")
+        lpdt = Client().get_data_type_info("litpop")
         self.assertEqual(lpdt.data_type, 'litpop')
         self.assertEqual(lpdt.data_type_group, 'exposures')
+        self.assertTrue('climada_version' in [p['property'] for p in lpdt.properties if p['mandatory']])
+        self.assertTrue('country_name' in [p['property'] for p in lpdt.properties if not p['mandatory']])
 
     def test_data_types(self):
         """"""
-        exdts = Client().get_data_types("exposures")
+        exdts = Client().list_data_type_infos("exposures")
         self.assertTrue('litpop' in [exdt.data_type for exdt in exdts])
 
     def test_datasets(self):
         """"""
-        datasets = Client().get_datasets(status=None, name='FAOSTAT_data_producer_prices')
+        datasets = Client().list_dataset_infos(status=None, name='FAOSTAT_data_producer_prices')
```

Comment:
Should not all datasets have a defined `status` ?

Reply:
Same as the property keys, the status is technically not restricted (yet) to a limited vocabulary. This provides a way to collect all datasets regardless of their status.